src/components/screens/DashboardGestaoProduto/index.jsx

```diff
@@ -1,22 +1,31 @@
 import React, { Component } from "react";
 import HTTP_STATUS from "http-status-codes";
-import { Link } from "react-router-dom";
+import CardStatusDeSolicitacao from "../../Shareable/CardStatusDeSolicitacao/CardStatusDeSolicitacao";
+import CardBody from "../../Shareable/CardBody";
+import { GESTAO_PRODUTO } from "../../../configs/constants";
+import { formataCards, getCardIcon, getCardStyle, getCardHREF } from "./helper";
+import { dataAtual, deepCopy } from "../../../helpers/utilities";
 
 export default class DashboardGestaoProduto extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      homologacoes: null,
+      dashboardData: null,
+      dashboardDataFiltered: null,
       erro: false
     };
+    this.onPesquisaChanged = this.onPesquisaChanged.bind(this);
   }
 
   componentDidMount() {
-    const { endpointGetHomologacoes } = this.props;
-    endpointGetHomologacoes()
+    const { endpointDashboard } = this.props;
+    endpointDashboard()
```

Comment:
Endpoint é so a URL. No caso como vc está passando uma função seria melhor nomear essa variavel com um nome mais propriado para uma função.  (sei que o projeto todo tá assim mais a idéia é ir aos poucos melhorando). 
Ex:
const {  fetchData } = this.props;
fetchData()

Reply:
beleza, alterei pra:
```
const { fetchDataDashboard } = this.props;
    fetchDataDashboard()
```